### ISSUE TITLE
Get the package listed in the API docs

### ISSUE THREAD

**Issue by maelle** [DoubleYouGTT/hubspot#13]
Once it is a bit more mature, obviously.

I don't know whether it's even possible to have it listed there but it'd be great.

**Comment by maelle** [comment]
Python package https://github.com/jpetrucciani/hubspot3

**Comment by maelle** [comment]
Mention in the forum of this package https://community.hubspot.com/t5/APIs-Integrations/R-API-Call/m-p/268738

**Comment by stephlocke** [comment]
Yeah Paul eventually made his way to us and got help using the package, should probably do some monitoring if possible!

**Comment by maelle** [comment]
I'll open a separate issue for that. :-)
